tests/model_serving/model_server/upgrade/test_upgrade_kserve_kueue_raw.py

```
@@ -0,0 +1,311 @@
+
+        status_replicas = None
+        try:
+            for status_replicas in TimeoutSampler(
+                wait_timeout=120,
+                sleep=5,
+                func=lambda: _get_deployment_status_replicas(deployment),
+            ):
+                if status_replicas == KSERVE_KUEUE_SCALED_REPLICAS:
+                    break
+        except TimeoutExpiredError:
+            assert False, (
```

Ruff (0.15.20)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)